### PR TITLE
[Release-1.5] Update istioctl manifest cni installation option

### PR DESCRIPTION
**Pull request by incfly**
it misses the `values.` prefix. invalid option.

### CODE REVIEW

File: content/en/docs/setup/additional-setup/cni/index.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ The following table shows the required settings for many common Kubernetes envir
     For example, the following `istioctl manifest` command sets the `cniBinDir` value for a GKE cluster:
 
     {{< text bash >}}
-    $ istioctl manifest apply --set cniBinDir=/home/kubernetes/bin
+    $ istioctl manifest apply --set values.cni.cniBinDir=/home/kubernetes/bin --set components.cni.enabled=true
```
Comment (jwendell) [inline content/en/docs/setup/additional-setup/cni/index.md]: You could update the sentence above this as well, no? 

Install Istio CNI via `Istioctl` including the `--set cniBinDir=/home/kubernetes/bin` option

Comment (incfly) [reply content/en/docs/setup/additional-setup/cni/index.md]: done.

Comment (ericvn) [inline content/en/docs/setup/additional-setup/cni/index.md]: The instructions a few lines up (#140) have a different --set for the cniBinDir.

The table above also says one needs to set the cni.namepsace and that's not included here.

Comment (incfly) [reply content/en/docs/setup/additional-setup/cni/index.md]: done.

**Comment by incfly** [comment]
@ericvn ptal, thx

**Pull request merged**